default/buildings.txt

```diff
@@ -1575,6 +1622,8 @@ BuildingType
         Population low = [[MIN_RECOLONIZING_SIZE]]
         OwnedBy empire = Source.Owner
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
I think it should be possible to avoid relying on references to specific named engine parts.  This restriction on colony pods seems quite a bit more complicated that the rest of this PR; I think it should be saved for a distinct PR.